src/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://www.mendix.com/package/1.0/">
-    <clientModule name="MicroflowTimer" version="3.1.1" xmlns="http://www.mendix.com/clientModule/1.0/">
+    <clientModule name="ActionTimer" version="3.2.0" xmlns="http://www.mendix.com/clientModule/1.0/">
```

Comment:
should be major update, as this version will only run on mx 7.13 and up

Reply:
Updated to 4.0.0